subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -391,6 +391,66 @@ uint8_t ll_adv_sync_param_set(uint8_t handle, uint16_t interval, uint16_t flags)
 		return err;
 	}
 
+#if defined(CONFIG_BT_CTLR_ADV_PDU_LINK)
+	struct pdu_adv *ter_pdu = (void *)lll_sync->data.pdu[ter_idx];
```

Comment:
Remember to remove this commit. I'll try to avoid approving the PR while it is still here.

Reply:
Noted :-)